website/source/guides/encryption/index.html.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: "guides"
+page_title: "Encryption as a Service - Guides"
+sidebar_current: "guides-encryption"
+description: |-
+  The transit secrets engine handles cryptographic functions on data in-transit.
+  Vault doesn't store the data sent to the secrets engine. It can also be viewed
+  as "cryptography as a service" or "encryption as a service".
+---
+
+# Encryption as a Service
+
+Vault provides Encryption as a Service (EaaS) to enables security teams to
+fortify data during transit and at rest. So even if an intrusion occurs, your
+data is encrypted with AES 256-bit CBC encryption (TLS in transit). Even if an
```

Comment:
Transit supports multiple cipher types (https://www.vaultproject.io/api/secret/transit/index.html#type). The default is the AES 256-bit GCM 96-bit. But it doesn't support AES in CBC mode.

Reply:
@vishalnayak  Thank you for the correction.  I think I copied that from some old doc.  I updated it to keep the intro simple.

Reply:
No problem!